### PR TITLE
docs: move "db transactions" to "how-to guides" in sidebar

### PR DESCRIPTION
**Pull request by bajtos**
This is a follow-up to #5718, see also #5549.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/guides/data-access/transactions.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ keywords:
   Transaction
 tags:
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Using-database-transactions.html
+permalink: /doc/en/lb4/data-access-guides-transactions.html
```
Comment (raymondfeng) [inline docs/site/guides/data-access/transactions.md:10]: Please capitalize the first letter: `Data-access-...`

File: docs/site/sidebars/lb4_sidebar.yml [hunk 2]
```diff
@@ -229,6 +229,10 @@ children:
           url: Where-filter.html
           output: 'web, pdf'
 
+    - title: 'Using Database Transactions'
+      url: Using-database-transactions.html
```
Comment (dhmlau) [inline docs/site/sidebars/lb4_sidebar.yml:233]: do you want to use the new html file name instead?

File: docs/site/guides/data-access/transactions.md (hunk 1 above)
Comment (bajtos) [reply docs/site/guides/data-access/transactions.md:10]: See https://github.com/strongloop/loopback-next/pull/5775#discussion_r443347719

I would like to transition our URL scheme to all-lowercase names and eventually to nested paths (`/reference/reserved-binding-keys.html`) to match the directory layout in `docs/site`.

We are already following the new style in migration guide, see e.g. here:

https://github.com/strongloop/loopback-next/blob/ae6427322451c914ae54f44dbb656981e7fbbb81/docs/site/migration/datasources.md#L7

Let's discuss in #5798

**Comment by bajtos** [comment]
Based on discussion with @raymondfeng, we will stick with Sentence-style.html to land my PRs and then normalize URL slugs afterwards, as this part may take a bit longer to reach consensus.

**Pull request merged**